### ISSUE TITLE
[Feature Request] Home Factions IDs in the populated systems nightly dump.

### ISSUE THREAD

**Issue by WarmedxMints** [EDSM-NET/FrontEnd#311]
Hello,

I am just wondering if it would be possible to add an array of id's for any factions which are native to the system in the nightly dump please?  Unless of course I am going blind and it is already in there :D

Thank you.

**Comment by klightspeed** [comment]
~~Try https://www.edsm.net/dump/systemsPopulated.json~~
Sorry, mis-read the request.

`controllingFaction` has the faction information for the controlling faction, while `factions` has the information on the factions present in the system.

**Comment by WarmedxMints** [comment]
I am aware, unfortunately `factions `doesn't contain their home system and nor does the general system information contain the factions which are native.  It would be great for me if that data could be included as it would save me from parsing two json files from eddb to get that info.

**Comment by klightspeed** [comment]
Ah ok - so something like an `isHomeSystem` flag on the faction in the `factions` array?

**Comment by WarmedxMints** [comment]
Yeah, a bool in the faction data would do it. I guess that would be less of an increase in data size than an int array of native factions ids in the `system` data.  I also forgot to mention that the unix time of the last update for the system would be nice. I assume that should be done in a new request though.

**Comment by klightspeed** [comment]
I suspect the faction update time being a unix time_t instead of an ISO datetime might be an oversight.

The `date` field in the system is when the system was last updated.  Note that updating a station, body or faction will not update this timestamp - it is strictly when the system name or location are updated.

**Comment by WarmedxMints** [comment]
Ok. thanks. That explain why the `date` field tends to be from a few years ago. I'll just use the main star update time field then to work out when the system was last updated.
